src/PrpShop/PRP/Surface/QMipmap.cpp

```diff
@@ -340,6 +353,17 @@ void QMipmap::saveDamage()
                 | (fFlags[kIsOrtho]->isChecked() ? plBitmap::kIsOrtho : 0));
 }
 
+static void swapColorChannels(unsigned char* data, size_t size)
+{
+    unsigned int* dp = reinterpret_cast<unsigned int*>(data);
```

Comment:
Just to ensure sanity, we should probably add an assert here that the data pointer is properly aligned, so the cast doesn't introduce undefined behavior.